Sources/Basic/OutputByteStream.swift

```diff
@@ -55,32 +58,108 @@ public class OutputByteStream: OutputStream {
         return buffer.count
     }
 
-    /// The contents of the output stream.
-    ///
-    /// This method implicitly flushes the stream.
-    public var bytes: ByteString {
-        flush()
-        return ByteString(self.buffer)
+    /// Currently available buffer size.
+    private var availableBufferSize: Int {
+        return max(buffer.capacity, OutputByteStream.bufferSize) - buffer.count
```

Comment:
I think instead of `max` here we should create the buffer with a fixed size (then it never needs to grow).

Otherwise this is kinda weird in that it can return a value greater than the capacity of the buffer, which makes it a misnomer.